GSA/digitalgov.gov
# USWDS Nov MC: Updating capitalization of Web Components in description

## Summary

Capitalizing Web Components to be consistent with industry standard.

## Preview

[Link to Preview]()




## How To Test

1. Navigate to events
2. Find USWDS November event
3. Confirm that Web Components is capitalized in 3rd bullet of description
